src/libsystemd/sd-bus/bus-socket.c

```diff
@@ -124,6 +124,8 @@ bool bus_socket_auth_needs_write(sd_bus *b) {
         return false;
 }
 
+static int bus_socket_auth_verify(sd_bus *b);
```

Comment:
You should just move the entire function here instead of forward declaring it

Reply:
Done. (I wanted to avoid big code movements for easier review.)